Supporting Documents/BS_SD.adoc

```diff
@@ -1749,7 +1749,7 @@ The evaluator shall take the following application notes into account to calcula
 
 ==== Application note for Elapsed time for Identification
 
-The evaluator shall select one week at maximum because the evalator shall finish the penetration testing within one week.
+The evaluator should select one week at maximum because the evalator should finish the penetration testing within one week for the testing of a single biometric sensor. This is not intended to force the evaluator to spend a week of time for testing each sensor type when multiple sensor types are tested in a single evaluation. The evaluator must be able to provide a sufficient justification that the time taken in testing for each sensor type is equivalent to a week for a single sensor test (when only a single sensor would be tested in isolation).
```

Comment:
I understand the motivation, but want to make sure we don't make this _too_ lax. What do we think about this modification?

```suggestion
The evaluator should select one week at maximum because the evalator should finish the penetration testing within one week for the testing of a single biometric sensor. This is not intended to force the evaluator to spend a week of time for testing each sensor type when multiple sensor types are tested in a single evaluation. The evaluator shall provide a sufficient justification that the time taken in testing for each sensor type is equivalent to a week for a single sensor test (when only a single sensor would be tested in isolation).
```